### ISSUE TITLE
About versions of VPoser

### ISSUE THREAD

**Issue by kklim810** [nghorbani/human_body_prior#46]
Hello.
Thank you for your wonderful work!

**I want to ask about the versions of this project.**

I just noticed the VPoser from SMPLify-X(CVPR_2019, Version 1) and from your master branch version(Version 2) are quite different.
The loss functions or defined joints for training are different and I think this will produce different results.
Did you update your version because the Version 2 produce quite better results?
I'm confused which version I should use.
Is there any plans to report results in papers with this new VPoser?

**Additionally, I want to ask about my trained results with version 2.**

I trained VPoser with your master branch, and nothing has been modified without debugging some errors.
I used your sample dataset which has about 500k poses for training.

These are the training log and the results of tutorials.
Because of the earlyStopping moddule, the epoch stops at 8.
![image](https://user-images.githubusercontent.com/80738757/142751429-bce55f9d-c6f0-46a7-8b95-0edd2cb80c8f.png)

1. From your pretrained model V02_05, the tutorial result which encode and decode sample pose is shown below.
![image](https://user-images.githubusercontent.com/80738757/142751478-579d4869-2e15-4e05-9f74-29b9a94cbcaa.png)

 

2. And my results shows that whatever pose it is, it shows the result of the arm being bent backwards.
![image](https://user-images.githubusercontent.com/80738757/142751501-1eddb74b-d7ba-43a2-9ec9-9c6c57cf1970.png)

I even tried by removing earlyStopping module and tried to train as much as I wanted, but there was still a problem.

I'd appreciate it if you could give me suggestion with it.

Thanks.

**Comment by keniel123** [comment]
hi is there an update on this ?
